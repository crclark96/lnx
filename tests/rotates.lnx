bl >> 3;
rcx << 7;
edx > 3;
eax < 0;
