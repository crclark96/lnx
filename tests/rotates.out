ror bl, 3
rol rcx, 7
shr edx, 3
shl eax, 0
